_bookdown.yml

```diff
@@ -37,5 +37,7 @@ rmd_files: ["index.Rmd",
             # "96_function_reference.Rmd",
             "97_extra_materials.Rmd",
 	    "90_acknowledgments.Rmd",
-            "99_bibliography.Rmd"
+	    "Session_info.Rmd",
```

Comment:
There shall be no tabs in yml files. I think this is likely to cause some errors/issues

-> remove tabs